memory/memhotplug.py

```diff
@@ -97,9 +106,16 @@ def setUp(self):
         self.stresstime = int(self.params.get('stresstime', default='10'))
         self.vmcount = int(self.params.get('vmcount', default='4'))
         self.iocount = int(self.params.get('iocount', default='4'))
+        self.hotplug_count = self.params.get('memcount', default='3')
```

Comment:
i think `self.hotplug_count` can be only zero or none zero value as i did not see any effect in code as it has any numeric value